src/js/letters/containers/Main.jsx

```diff
@@ -10,9 +12,19 @@ class Main extends React.Component {
   }
 
   render() {
+    let appContent;
+
+    if (this.props.lettersAvailable) {
+      appContent = this.props.children;
+    } else if (this.props.lettersAvailable === undefined) {
```

Comment:
Yeah, something like that. The checks should still be pretty clear, since you should be making comparisons like `this.props.letterStatus === LETTER_STATUSES.available` (or however you want to create constants for the different statuses).